### PR TITLE
remove bluebird.each

### PR DESCRIPTION
**Pull request by maximelkin**
closes https://github.com/tgriesser/knex/issues/3256

### CODE REVIEW

**Comment by maximelkin** [comment]
@kibertoad maybe its better to merge all prs in one?

File: lib/dialects/oracledb/index.js [hunk 1]
```diff
@@ -128,7 +129,7 @@ Client_Oracledb.prototype.acquireRawConnection = function() {
           });
         });
       };
-      const fetchAsync = function(sql, bindParams, options, cb) {
+      const fetchAsync = promisify(function(sql, bindParams, options, cb) {
```
Comment (kibertoad) [inline lib/dialects/oracledb/index.js]: Does this function use any closure variables, or it can be moved outside to only be promisified once?

**Comment by kibertoad** [comment]
@maximelkin Keeping PRs separate makes them easier to review :)

File: lib/dialects/oracledb/index.js (hunk 1 above)
Comment (maximelkin) [reply lib/dialects/oracledb/index.js]: This function take connection from outside

**Pull request merged**